### PR TITLE
260a image uploads

### PR DESCRIPTION
**Pull request by Lepozepo**

### CODE REVIEW

**Comment by newspringweb** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/rock/models/binary-files/model.js [hunk 1]
```diff
@@ -55,10 +57,22 @@ export class BinaryFile extends Rock {
...
-
+  async attachPhotoIdToUser({ personId, previousPhotoId, newPhotoId } = {}) {
+    try {
+      await api.patch(`/People/${personId}`, {
+        PhotoId: newPhotoId,
+      });
+      if (!isEmpty(previousPhotoId)) {
+        try {
+          await api.delete(`/BinaryFiles/${previousPhotoId}`);
+        } catch (e) {} // eslint-disable-line
```
Comment (IsaacHardy) [inline src/rock/models/binary-files/model.js:70]: I'm speaking out of ignorance here, but can this `.delete()` method return an error that show be handled?

Comment (Lepozepo) [reply src/rock/models/binary-files/model.js:70]: @IsaacHardy I agree with you but I'm not sure how we would handle it. This is coming (partially) from [here](https://github.com/NewSpring/Holtzman/blob/master/imports/deprecated/methods/files/server/avatar.js#L31)

Comment (IsaacHardy) [reply src/rock/models/binary-files/model.js:70]: Ah, I see. I'm going to ask Rich to see if he knows how we might be able to handle that potential catch.

**Comment by Lepozepo** [comment]
@conrad-vanl this is the Heighliner counterpart to [this](https://github.com/NewSpring/Apollos/pull/294) and is ready for review as well ^_^

**Review by conrad-vanl — APPROVED** [review]

**Pull request merged**